### PR TITLE
Added version handling for RHACS docs

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Adds the backend code for handling version selector click events for RHACS docs.

- Simplified the checking of versions by using an array prototype function.
- Added logic to handle RHACS versions.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@aireilly Can you please review this PR?

**Comment by aireilly** [comment]
How can I build this locally to test? I'm unfamiliar with the RHACS build

**Comment by aireilly** [comment]
LGTM. Built and tested locally, page selector + EOL banner works as described.

File: _javascripts/page-loader.js [hunk 1]
```diff
@@ -57,7 +67,10 @@ function versionSelector(list) {
       if(jqXHR.status == 404) {
         list.value = currentVersion;
         if(confirm("This page doesn't exist in version " + newVersion + ". Click OK to search the " + newVersion + " docs OR Cancel to stay on this page.")) {
-          window.location = "https://google.com/search?q=site:https://docs.openshift.com/container-platform/" + newVersion + " " + document.title.split(" | ")[0];
+          if (['3.65', '3.66', '3.67', '3.68', '3.69', '3.70', '3.71'].contains(newVersion)) {
+          window.location = "https://google.com/search?q=site:https://docs.openshift.com/acs/" + newVersion + " " + document.title;} else {
+            window.location = "https://google.com/search?q=site:https://docs.openshift.com/enterprise/" + newVersion + " " + document.title;
```
Comment (gaurav-nelson) [inline _javascripts/page-loader.js:72]: @adellape This line 72 is what we have in the `main` branch, however, we have a different line in `enterprise-4.1` branch at https://github.com/openshift/openshift-docs/blob/enterprise-4.1/_javascripts/page-loader.js#L60

I am leaning towards keeping L60 from `enterprise-4.1` branch and then also cherry-picking these changes to the main branch. Can you look at this and suggest?

Comment (adellape) [reply _javascripts/page-loader.js:72]: Considering https://github.com/openshift/openshift-docs/pull/39658, I think that makes sense.

**Comment by aireilly** [comment]
LGTM

**Pull request merged**